tests/test_integrated_channels/test_integrated_channel/test_exporters/test_course_metadata.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+"""
+Tests for the base course metadata exporter.
+"""
+
+from __future__ import absolute_import, unicode_literals, with_statement
+
+import unittest
+
+import mock
+import responses
+from integrated_channels.integrated_channel.exporters.course_metadata import CourseExporter
+from pytest import mark
+
+from test_utils import factories
+from test_utils.fake_enterprise_api import EnterpriseMockMixin
+
+
+@mark.django_db
+class TestCourseExporter(unittest.TestCase, EnterpriseMockMixin):
+    """
+    Tests for the ``CourseExporter`` class.
+    """
+
+    def setUp(self):
+        self.enterprise_customer = factories.EnterpriseCustomerFactory()
+        self.mock_ent_courses_api_with_pagination(
+            enterprise_uuid=str(self.enterprise_customer.uuid),
+            course_run_ids=['course-v1:edX+DemoX+Demo_Course_1']
+        )
```

Comment:
@UmanShahzad Same note here about moving this down to keep mock definitions together.

Reply:
:+1: